.github/workflows/build-plugin-archive.yml

```
@@ -34,7 +34,7 @@ on:
         required: false
         type: string
       ARCHIVE_NAME:
-        description: The base name of the resulting zip archive. Falls back to the repository name.
+        description: The full name of the resulting zip archive. Falls back to the repository name.
```

Let's use the same "syntax" as here: https://github.com/inpsyde/reusable-workflows/blob/11582432bb2a4b4fdb793823a22f4b2a1d2267c1/.github/workflows/automatic-release.yml#L12C1-L12C1.
```suggestion
        description: The name of the zip archive (falls back to the repository name).
```